scripts/test_robusta_platform_mcp.py

```
@@ -0,0 +1,138 @@
+            logger.warning("expected isError for account with no Slack: %s", payload)
+            raise RuntimeError("expected isError but got success")
+        if "no Slack integration" not in payload:
+            logger.warning("unexpected error payload: %s", payload)
+            raise RuntimeError("unexpected error payload")
+        logger.info("[ok] negative path: %s", payload)
+
+
+if __name__ == "__main__":
+    try:
+        main()
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)